src/components/PortfolioAccordions.vue

```diff
@@ -38,46 +37,47 @@
         </div>
       </div>
 
-      <!-- Use the accurate heading level to maintain the document outline -->
-      <h4 class="usa-accordion__heading">
+
+     <!-- Use the accurate heading level to maintain the document outline -->
+      <h3 class="usa-accordion__heading">
         <button
           class="usa-accordion__button"
           aria-expanded="false"
           aria-controls="m-a2"
         >
-          River Conditions
+          Chart Gallery
         </button>
-      </h4>
+      </h3>
       <div
-        id="m-a2"
+        id="m-a4"
```

Comment:
doesn't this need to be `id="m-a2"`? As-is, the accordion doesn't collapse the content in when you click it